blas/testblas/common.go

```diff
@@ -260,7 +260,16 @@ func zsame(x, y []complex128) bool {
 	}
```

Comment:
Can we have an explanation in the commit message for this for why the change was necessary.

Reply:
Done.